apiserver/plane/api/views/oauth.py

```diff
@@ -108,6 +107,12 @@ def get_user_data(access_token: str) -> dict:
 
     userData = resp.json()
 
+    req = requests.get(url="https://api.github.com/user/emails", headers=headers)
+    resp = req.json()
```

Comment:
please rename variable to `response` and add `.json()` after `)`  which will help omit line 111